code/src/index.css

```diff
@@ -1,13 +1,45 @@
+/** @format */
+
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
+
+html {
+	background-color: #212529;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: "Press Start 2P", cursive, -apple-system, BlinkMacSystemFont,
+		"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
+		"Droid Sans", "Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
+}
+
+.display {
+	text-align: center;
+	margin: auto;
+	background-color: #212529;
+	padding: 40px;
+	font-size: 14px;
+}
+
+.input-text {
+	font-size: 10px;
+}
+
+.loading {
+	color: white;
+}
+
+@media (min-width: 767px) {
```

Comment:
One suggestion could be to change the max-width for nes-container, something like 800px, so that it is not so wide on the big screen 🕵️‍♀️